src/multi.c

```diff
@@ -36,6 +36,7 @@ void initClientMultiState(client *c) {
     c->mstate.commands = NULL;
     c->mstate.count = 0;
     c->mstate.cmd_flags = 0;
+    c->mstate.cmd_inv_flags = 0;
```

Comment:
We don't just use ~cmd_flags? The use of an additional field looks a bit too much to me.

Reply:
Nevermind, here you want to be sure that all the commands have a specific flag.